.github/workflows/django-build.yml

```diff
@@ -0,0 +1,44 @@
+name: Django Build
+
+on:
+  push:
+    branches: [main]
+  pull_request:
+    branches: [main]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      max-parallel: 4
+      matrix:
+        python-version: [3.9]
+
+    services:
+      mysql:
+        image: mysql:5.7
+        env:
+          MYSQL_ROOT_PASSWORD: joladnijo
+          MYSQL_DATABASE: joladnijo
+        ports: ["3306:3306"]
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up Python ${{ matrix.python-version }}
+        uses: actions/setup-python@v2
+        with:
+          python-version: ${{ matrix.python-version }}
+      - name: Install Dependencies
+        run: |
+          python -m pip install --upgrade pip
+          pip install -r requirements.txt
+      - name: Run Migrations
+        run: python manage.py migrate
+        env:
+          DBENGINE: django.db.backends.mysql
+          DBNAME: joladnijo
+          DBUSER: root
+          DBPASSWORD: joladnijo
+          DBHOST: 127.0.0.1
+          DBPORT: 3306
+          SECRET_KEY: "jqa&w(c3sdhdxzafa)c0u71c%@v()vxzst!y^v*9^!-)&ouhp9"
```

Comment:
Ez mehet GitHub Action secretbe. Adok admin jogot a repohoz, hogy hozza tudd adni.

Reply:
Vagy lehet, hogy ez nem is kell, ha csak a build idejen van hasznalatban.

Reply:
Ez csak itt a build pipeline-ban van. Kicserélem ezt is valami feltűnően nem titkosra.